capstone/src/main/java/com/google/sps/servlets/LoadDataServlet.java

```diff
@@ -69,4 +84,31 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     response.setContentType("application/json;");
     response.getWriter().println(gson.toJson(surveyResponses));
   }
+
+  public void getPredictions(
+      HttpServletRequest request, HttpServletResponse response, PreparedQuery results)
+      throws IOException {
+    String directExperience = request.getParameter("directExperience");
+    String gender = request.getParameter("gender");
+    String ageRange = request.getParameter("ageRange");
+    float score = 0;
+
+    for (Entity e : results.asIterable()) {
+      String entityDirectExp = (String) e.getProperty("directExperience");
+      String entityGender = (String) e.getProperty("gender");
+      String entityAgeRange = (String) e.getProperty("ageRange");
+
+      if (entityGender.equals(gender)
+          && entityAgeRange.equals(ageRange)
+          && entityDirectExp.equals(directExperience)) {
+        score = (float) (double) e.getProperty("score");
```

Comment:
Do you need to cast twice here?

Reply:
I believe so, it seems like I cannot convert a data store property directly to a float (after reviewing Prashanti's PR)